samples/SampleOrigin/wwwroot/Test.htm

```diff
@@ -0,0 +1,137 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <style>
+        .success {
+            background-color: #4CAF50;
+        }
+    </style>
+    <title>CORS Sample</title>
+</head>
+<body>
+    <p>CORS Sample</p>
+    <table>
+        <thead>
+            <tr>
+                <th>Scenario</th>
+                <th>Result</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr id="scenarios"></tr>
+        </tbody>
+    </table>
+    <script type="text/javascript">
+        (function () {
+            function _fetch(options) {
+                const url = 'http://localhost:9000/' + options.testPath;
+                delete options.testPath;
+                options.cors = 'cors';
+
+                return fetch(url, options);
+            }
+
+            function assertSuccess(options) {
+                return _fetch(options).then(s => true).catch(s => false);
```

Comment:
What is the error that you get when the request fails. Is there a way to be more specific?